### PR TITLE
Remove checked-in configuration

### PR DESCRIPTION
**Pull request by danielocfb**
Both the bpf and bpf-next tree have suitable BPF selftest configurations
available for usage with the latest kernel now upstream. While we do
test on 4.9 and 5.5 kernels as well, there we just download prebuilt
binaries. The configuration we use for building selftests is always the
upstream one.
With this change we remove the checked-in configuration, as it is now no
longer needed.

Signed-off-by: Daniel Müller <deso@posteo.net>

### CODE REVIEW

**Comment by danielocfb** [comment]
This pull request is kind of a follow up to https://github.com/libbpf/libbpf/pull/537. I wanted to land that one first to make sure that no issues occur, but conceptually they could have been a single change as well.

File: .github/actions/vmtest/action.yml [hunk 1]
```diff
@@ -41,12 +41,10 @@ runs:
     - name: Prepare to build BPF selftests
       shell: bash
       run: |
-        echo "::group::Prepare buidling selftest"
+        echo "::group::Prepare buildling selftest"
```
Comment (mykola-lysenko) [inline .github/actions/vmtest/action.yml:44]: building?

File: .github/actions/vmtest/action.yml [hunk 2]
```diff
@@ -41,12 +41,10 @@ runs:
...
         cd .kernel
         cat tools/testing/selftests/bpf/config \
-            tools/testing/selftests/bpf/config.${{ inputs.arch }} \
-            ${{ github.workspace }}/travis-ci/vmtest/configs/config-latest \
-            ${{ github.workspace }}/travis-ci/vmtest/configs/config-latest.${{ inputs.arch }} 2> /dev/null > .config && :
+            tools/testing/selftests/bpf/config.${{ inputs.arch }} > .config
         make olddefconfig && make prepare
         cd -
         echo "::endgroup::"
```
Comment (mykola-lysenko) [inline .github/actions/vmtest/action.yml:50]: not related to this change. can we reuse helpers to start / end group in here?

File: .github/actions/vmtest/action.yml (hunk 1 above)
Comment (danielocfb) [reply .github/actions/vmtest/action.yml:44]: Oh yeah, not sure what happened here.

File: .github/actions/vmtest/action.yml (hunk 2 above)
Comment (danielocfb) [reply .github/actions/vmtest/action.yml:50]: I believe we can, yes. I'll do it as a follow up change.

**Comment by danielocfb** [comment]
I've addressed the review comment.

**Review by mykola-lysenko — APPROVED** [review]

**Pull request merged**